gui/src/app/init.js

```diff
@@ -820,16 +821,10 @@ class GUIApp {
       return;
     }
     console.log('successfully logged into controller');
-
-    // If the GUI is embedded in storefront, we need to share login
-    // data with the storefront backend and ensure we're already
-    // logged into the charmstore.
     if (this.applicationConfig.gisf) {
-      this._sendGISFPostBack(() => {
-        // We can only request the updated config file after the storefront has
-        // been notified of the users login information.
-        this.reloadConfigFile(document);
-      });
+      // If this is the JAAS GUI then we need to set a cookie to say that we're
+      // logged in to correctly redirect requests to jaas.ai if necessary.
+      cookie.set('loggedin', 'true');
```

Comment:
Did you consider this format? Is naming cookies lowercase without spaces a common standard? If not I think we should consider it. We should either follow the precedent in this app of lowercase_with_underscores (`auth_tkt`) or come up with a new standard deliberately.

Personally, I would choose lowercase-with-hyphens - so `logged-in`. This [seems like it should work everywhere](https://stackoverflow.com/a/1969339/613540), and is the standard we use for most things that aren't actually programming - URL paths, class names and IDs, etc.